python/lsst/ts/ofc/ofc.py

```diff
@@ -151,13 +151,13 @@ def calculate_corrections(self, wfe, field_idx, filter_name, gain, rot):
         else:
             self.ofc_controller.gain = gain
 
-        optical_state = self.state_estimator.dof_state(filter_name, wfe, field_idx)
+        optical_state = self.state_estimator.dof_state(filter_name, wfe, field_idx, rot)
```

Comment:
It is consistent. The rotation angle is passed in this function to compute the rotated sensitivity matrix and that is done only when we use the Double_zernikes attribute. Otherwise (when DZ is not enabled), this argument here is not used. When DZ is not enabled we rotate the degree of freedom state. When it is enabled we only rotate the sensitivity matrix. Let me know if you need further clarification.

Reply:
In this case, you should explain this argument and related detail in the doc string of `StateEstimator.dof_state()`. In addition, it sounds like this should be an `optional` argument.